boards/BMS/Src/main.c

```diff
@@ -972,7 +974,7 @@ void RunTaskCanTx(void const *argument)
 
     for (;;)
     {
-        Io_SharedCan_TransmitEnqueuedCanTxMessagesFromTask();
+        Io_SharedCan_TransmitEnqueuedCanTxMessagesFromTask(error_table);
```

Comment:
I think it is better to set BMS errors through the state machine when we detect it. `Io_SharedErrorTable_SetErrorsFromCanMsg` is better suited for setting the error table from received CAN messages

Reply:
I know, this method calls `Io_SharedErrorTable_SetErrorsFromCanMsg` if you look down below.  I haven't called it in the `RunTask___` method like I did for CanRx because the CAN message needs to get popped off the tx queue first, and this is done in `Io_SharedCan_TransmitEnqueuedCanTxMessagesFromTask`